.github/workflows/build-test-push.yml

```diff
@@ -7,7 +7,7 @@ on:
     branches:
       - master
 jobs:
-  if: ${{ !startsWith(github.event.head_commit.message, 'v202') && !contains(github.event.head_commit.message, 'skip CI') }}
+  # if: ${{ !startsWith(github.event.head_commit.message, 'v202') && !contains(github.event.head_commit.message, 'skip CI') }}
```

Comment:
I think this change should be reverted. The intent of the conditional is to prevent yet another regeneration of binaries after [these steps](https://github.com/ampproject/amp-closure-compiler/blob/master/PUBLISH.md#publishing-a-new-version-of-ampprojectamp-closure-compiler) have been run to update the version.

Reply:
Without commenting this line out, the Github Actions fail to start on this PR. Perhaps there is a tweak that satisfies both conditions?

Reply:
That seems unexpected. Is this a conventional PR from a fork?

Reply:
This PR is a bump of the version number for `closure-compiler` directly (not from renovate).

When building locally, everything works and symlinking (via npm link) works as expected with `amphtml`.

Reply:
I now see what you're seeing.

![image](https://user-images.githubusercontent.com/26553114/98280544-8f3b7e00-1f69-11eb-89a9-c3c02d6cb929.png)

I'm happy to fix this via a separate PR. Meanwhile, feel free to comment out this line and make progress.

We'll need to manually cancel the jobs that come up during publishing, I can do that for you once this PR is merged.

Reply:
I just pushed a commit to this PR to fix the conditional. Looks like the `if` clause needed to be under `build` and not `jobs`. The checks are running now.